client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
+import analytics from 'lib/analytics';
+
+function redirectToPartner() {
+	if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+		analytics.tracks.recordEvent( 'calypso_triforce_partner_redirect', { partner_name: 'Bluehost with WooCommerce' } );
```

Comment:
Nice improvement @vindl, but we have Redux actions for these so that we never have to call `analytics.x.y` in the components…

```js
import { recordTracksEvent } from 'state/analytics/actions';

something {
	const { recordPartnerRedirect } = this.props;

	recordPartnerRedirect( 'Bluehost with WooCommerce' );
}

// or

const redirectToPartner( recorder ) {
	recorder( 'Bluehost with WooCommerce' );
}

something {
	redirectToPartner( this.props.recordPartnerRedirect );
}

const mapDispatchToProps = dispatch => ( {
	recordPartnerRedirect: partner_name => dispatch( recordTracksEvent( 'calypso_triforce_partner_redirect', { partner_name } ) )
} );

export default connect( null, mapDispatchToProps )( localize( BluehostStoreStep ) );
```